src/functions/defaultMessage.ts

```diff
@@ -0,0 +1,10 @@
+import { Handler } from 'aws-lambda';
+import { response } from '../utils/response';
+import { LambdaResponse } from '../types';
+
+/**
+ * Default handler.
+ */
+export const handler: Handler = async (): Promise<LambdaResponse> => {
+  return response(200, 'Default message');
+};
```

Comment:
add console.log(event.body);
Would be helpful in testing future messages that specified by a route t hat hasn't been implemented.

Reply:
sure